src/app/services/news.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { News } from '../models/News';
+
+export interface NewsService {
+  getNews(): Observable<News[]>;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DummyNewsService implements NewsService {
+  getNews(): Observable<News[]>  {
+    return from([[
+      {
+        url: 'http://www.google.com',
+        title: 'COVID kills',
+        description: 'Dumb ppl still not wearing masks.'
+      },
+      {
+        url: 'http://www.bridgebase.com/v3',
+        title: 'Moron dies after injecting himself with bleach',
+        description: 'Trump claims injecting bleach as a COVID treatment is "very, very bad"'
+      }
+    ]]);
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ActualNewsService implements NewsService {
+  url = 'https://api.smartable.ai/coronavirus/news/';
+  subscriptionKey = '955eacf9525e45dd8d46a07b7daa649e';
+  region = 'global';
```

Comment:
Could enumerate the regions, letting you loop over them in an angular component as a dropdown or something, and then take a region parameter on `getNews`? Default it to global if no parameter is provided or something

Reply:
In another PR (#19) I've used a package (`iso-3166-1`) which has all country info. We should either spread to using that everywhere or switch to something else and I can change to using what we pick.

It contains `Country` objects which have their name, and several codes (it seems like the alpha2 codes are the most used in apis). We'll just have to have something to identify `global` then.

Reply:
Actually just seen your region service in #17, i think that's the way to go, and using the package through that service instead? I'll leave some comments there since switching to that is cleaner

Reply:
Using a package with the country info is definitely a good idea. Better than typing it by hand anyway!